sources/au/victoria.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "http",
-    "data": "http://s3.amazonaws.com/data.openaddresses.io/cache/au-victoria.zip",
-    "website": "http://services.land.vic.gov.au/landchannel/content/vicmapdata?productID=1",
+    "data": "https://www.alantgeo.com.au/share/au-victoria.zip",
```

Comment:
Yea, that's a curiosity with GitHub's OAuth system. We have to use these permissions to determine if you're a contributor to openaddresses' GitHub team.

If you're not comfortable, I will reupload it when I get back home.